blackbox/docs/configuration/cluster.txt

```diff
@@ -634,12 +634,12 @@ across generic attributes associated with nodes.
 
   Define node attributes which will be used to do awareness based on the
   allocation of a shard and its replicas. For example, let's say we have
-  defined an attribute ``rack_id`` and we start 2 nodes with ``node.rack_id``
-  set to rack_one, and deploy a single table with 5 shards and 1 replica. The
-  table will be fully deployed on the current nodes (5 shards and 1 replica
-  each, total of 10 shards).
+  defined an attribute ``rack_id`` and we start 2 nodes with
+  ``node.attr.rack_id`` set to rack_one, and deploy a single table with 5
```

Comment:
Should we also add this setting to the configuration page?

Reply:
there's a lot of stuff we need to improve in the docs re *what* is documented. but I'd prefer to tackle them in a different pr